File: library/src/main/java/com/juanimoli/strictmodenotifier/detector/ClassInstanceLimitDetector.java
package com.juanimoli.strictmodenotifier.detector;

import com.juanimoli.strictmodenotifier.StrictModeLog;

public class ClassInstanceLimitDetector implements Detector {

    @Override
    public boolean detect(StrictModeLog log) {
        return log.message.contains("StrictMode.setClassInstanceLimit");
    }
}

File: notifier/src/main/java/io/github/juanimoli/strictmode/notifier/detector/ClassInstanceLimitDetector.kt
package io.github.juanimoli.strictmode.notifier.detector

import io.github.juanimoli.strictmode.notifier.StrictModeLog

class ClassInstanceLimitDetector : Detector {
    override fun detect(log: StrictModeLog): Boolean {
        return log.message.contains("StrictMode.setClassInstanceLimit")
    }
}
